crate/operator/handlers/handle_update_user_password_secret.py

```diff
@@ -33,6 +36,19 @@ async def update_user_password_secret(
     diff: kopf.Diff,
     logger: logging.Logger,
 ):
+    # extract cluster_id from the secret name which can be either
+    # `user-<cluster_id>` or `user-password-<cluster_id>-<ordinal>`
+    cluster_id = re.sub(r"^user-(?:password-)?", "", name)
```

Comment:
yes, in newly created projects, the secret is called ``user-password-{cluster_id}-{ordinal}`` but it could be that in older projects it is still ``user-{cluster_id}`` ?

Reply:
Worth checking if we have any like that in prod anywhere?

Reply:
Sure, will do.